qt/python/mantidqt/test/test_algorithm_dialog.py

```diff
@@ -0,0 +1,59 @@
+import unittest
+
+from qtpy.QtWidgets import QWidget, QLineEdit
+
+from mantid.api import PythonAlgorithm
+from mantid.kernel import Direction, FloatArrayProperty
+from mantid.simpleapi import AlgorithmManager, AlgorithmFactory
+from mantidqt.utils.qt.plugins import setup_library_paths
+from mantidqt.utils.qt.testing import gui_test_case
+
+from _widgetscoreqt5 import AlgorithmDialog, GenericDialog, InterfaceManager
```

Comment:
We shouldn't import from `_widgetscoreqt5` directly as this package will need to support Qt4 & Qt5 later. Please add a thin wrapper module as is done in `messagedisplay.py`

Reply:
I thought the new workbench was Qt5 only

Reply:
It will be but the `mantidqt` package will need to be used from other code that still uses Qt4 so I'd like to keep this one Qt4 compatible. I think it will also prevent headaches in the future when moving forward with Qt versions.